packages/repack/src/modules/ScriptManager/__tests__/ScriptManager.test.ts

```diff
@@ -590,7 +597,7 @@ describe('ScriptManagerAPI', () => {
     expect(NativeScriptManager.loadScript).toHaveBeenCalledTimes(1);
     expect(NativeScriptManager.loadScript).toHaveBeenCalledWith(scriptId, {
       absolute: false,
-      fetch: false,
+      fetch: true,
```

Comment:
why do we need to set `fetch` to true here?

Reply:
This is the main problem This PR solves. If a LoadScript failed, it should be fetched again.
Currently, repack is caching even loadScript failed.